src/Uncontrolled.js

```diff
@@ -44,13 +42,11 @@ Object.keys(components).forEach((key) => {
 const UncontrolledAlert = components.UncontrolledAlert;
 const UncontrolledButtonDropdown = components.UncontrolledButtonDropdown;
 const UncontrolledDropdown = components.UncontrolledDropdown;
-const UncontrolledNavDropdown = components.UncontrolledNavDropdown;
 const UncontrolledTooltip = components.UncontrolledTooltip;
 
 export {
   UncontrolledAlert,
   UncontrolledButtonDropdown,
   UncontrolledDropdown,
-  UncontrolledNavDropdown,
```

Comment:
Would be nice to deprecate this the same as `NavDropdown` so that it is not a breaking change.